tools/opt/size-passes.h

```diff
@@ -0,0 +1,24 @@
+#ifndef SPIRV_TOOLS_SIZE_PASSES_H_
+#define SPIRV_TOOLS_SIZE_PASSES_H_
+
+#include <vector>
+#include <string>
+
+namespace spvtools {
+
+// Size optimization flags expanded by -Os.
+//
+// Using the -Os flag in spirv-opt's command line is equivalent to specifying
+// the following flags in this exact sequence.
+std::vector<std::string> SizePasses = {
```

Comment:
Make a function to return this.  

"Variables of class type with static storage duration are forbidden: they cause hard-to-find bugs due to indeterminate order of construction and destruction"

Reply:
Done.  I've reworked the whole -O and -Os handling.  They are now part of the Optimizer API. There are two new methods: RegisterPerformancePasses and RegisterSizePasses.  These do the direct instantiation of the related passes.  Having them inside the Optimizer API allows us to invoke them from places other than spirv-opt.